src/command_bar.rs

```diff
@@ -372,6 +372,33 @@ impl CommandBar {
                     self.show_hint();
                 }
             }
+            KeyCode::Delete => {
+                if self.command_line.chars().count() == 0 {
+                    self.show_hint();
+                }
+
+                self.update_command_list();
```

Comment:
This line updates the command list before the change of command bar text. This bug also occurs in `Backspace` handling.